frontend/package.json

```diff
@@ -98,7 +100,7 @@
     "format": "concurrently \"npx prettier --write .\" \"node_modules/.bin/next lint --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"",
     "serve": "next start",
     "test": "playwright test",
-    "e2e": "playwright test",
```

Comment:
This may be overwritten here: https://github.com/chanzuckerberg/single-cell-data-portal/blob/483782a8d3880056409637c8c40a0d5ec9313c7f/frontend/playwright.config.ts#L12

We probably want to keep GitHub reporter as that's giving us the nice summaries too.